pom.xml

```diff
@@ -21,7 +41,30 @@
     <dependency>
       <groupId>com.apifest</groupId>
       <artifactId>apifest-api</artifactId>
-      <version>0.1.1</version>
+      <version>0.1.2-SNAPSHOT</version>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.module</groupId>
+      <artifactId>jackson-module-jaxb-annotations</artifactId>
+      <version>2.5.0</version>
+    </dependency>
+    <!-- Test Scope -->
+    <dependency>
+      <groupId>org.testng</groupId>
+      <artifactId>testng</artifactId>
+      <version>6.8.21</version>
```

Comment:
That version of the library requires Java 8. The ApiFest project is supposed to run with Java 6+ so please change the version.